.github/workflows/build.yml

```diff
@@ -0,0 +1,25 @@
+name: Build
+
+on:
+  push:
+  pull_request:
+  release:
+    types: [ published ]
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        php: [ '5.6', '7.0', '7.1', '7.2' ]
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install composer
+        uses: php-actions/composer@v5
```

Comment:
This is the only change from the previous PR. For some reason the "dynamic-docker" version has started causing issues now. So this is a safer alternative. The remaining changes are the same as on [#36](https://github.com/Bynder/bynder-php-sdk/pull/36) 

Also, please ignore the failed checks under this PR as that's what I intend to fix 😄  with this PR.